docs/administrator/security/component-permission.md

```diff
@@ -0,0 +1,73 @@
+---
+title: Component Permissions
+---
+
+# Permissions for Karmada Components
+
+This document provides a detailed explanation of the Role-Based Access Control (RBAC) permissions required for key components of Karmada. It aims to ensure secure management and compliant access to resources within the system. As a multi-cluster management framework, controlling permissions between Karmada components is essential for maintaining the security and stability of the entire system. This guide will help administrators understand and configure the RBAC permissions needed for Karmada components effectively.
+
+For more detailed information about RBAC permissions, please refer to the [official RBAC documentation](https://kubernetes.io/docs/reference/access-authn-authz/rbac/).
+
+*Note: The resources in this document are all Karmada control plane resources.*
+
+## karmada-operator
+
+### Component Description:
+
+The Karmada operator is a method for installing, upgrading, and deleting Karmada instances. It builds upon the basic Karmada resource and controller concepts, provides convenience to centrally manage entire lifecycle of Karmada instances in a global cluster.
+
+### Component Permissions:
+
+#### Cluster-Scope:
+
+| Resource        | API Group           | Resource Names | Verbs                       | Description                                                  |
+| --------------- | ------------------- | -------------- | --------------------------- | ------------------------------------------------------------ |
+| leases          | coordination.k8s.io | /              | get, create, update         | Required for leader election                                 |
```

Comment:
What does the `/` mean? mean `all` or `NA`?

Reply:
it means `all`. Should I change `/` to `all`?

Reply:
`*` would be better